setup.py

```diff
@@ -19,12 +19,16 @@
     include_package_data=True,
     zip_safe=False,
     install_requires=[
-        'clld', 'clldmpg>=3.1.1',
+        'clld~=4.0',
+        'clldmpg>=3.1.1',
+        'sqlalchemy',
+        'waitress',
     ],
     extras_require={
         'dev': ['flake8', 'waitress'],
         'test': [
             'mock',
+            'tox',
```

Comment:
I would add `tox` to the dev requirements, since it's only needed for local development, while travis uses its own mechanism to provide cross-platform testing.